<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_create">
    <property name="geometry">
     <rect>
      <x>359</x>
      <y>50</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans T Chinese Medium</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>create</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_num">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>49</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans T Chinese Medium</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>How many?</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_result">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>88</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans T Chinese Medium</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>resultï¼š</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEdit_num">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans T Chinese Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="result_print">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>411</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans T Chinese Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtn_repeat">
    <property name="geometry">
     <rect>
      <x>217</x>
      <y>50</y>
      <width>120</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans T Chinese Medium</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Allow repeat</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans T Chinese Medium</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_from">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>13</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans T Chinese Medium</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_to">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>13</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans T Chinese Medium</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>to</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEdit_to">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>13</y>
      <width>170</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans T Chinese Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEdit_from">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>13</y>
      <width>170</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans T Chinese Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>455</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>Create random numbers</string>
    </property>
   </widget>
   <addaction name="menuCreate"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
